ccd-definition/ComplexTypes/SmallClaimsHearing-SDO-HNL-nonprod.json

```diff
@@ -16,16 +16,47 @@
   {
     "ID": "SmallClaimsHearing",
     "ListElementCode": "time",
-    "FieldType": "FixedList",
+    "FieldType": "FixedRadioList",
```

Comment:
Should a new complex type have been created so as not to break existing cases?

Reply:
after some thought i don't think this will be a problem, because it's saved as the same type it's just shown in a different format. if a case were to come back to the same case, and it has old data, it would just pick/display the old data chosen in the radio format/list format. happy to discuss this further.